### PR TITLE
"Meta" milestone

### LINKED ISSUE #14: GAME_DEBUG define as compile flag
Right now we have GAME_DEBUG flag hardcoded inside source files.
It's changes results in git. To make a prod you need to switch it's value on and off.

Better idea is to move this flag to compile flags.

### LINKED ISSUE #43: Determine license
There are some options. MIT, BSD, Unlicense or GPL. Also, this decision may affect our prods' licenses.

## Viral vs Non-viral (GPL)

Pros:

- It would be cool to have more and more open source stuff on Amigas. GPL would enforce it.

Cons:

- People may prefer using other dev tools which don't enforce opening their sources.
- There are some ppl in retrodev who'd like to keep their sources closed
- KaiN hates those mile-long comments on top of every source file
- If we get other contributors that would mean tracing fn history in their respective comment blocks, so that each contributor gets credit for their work

## Attribution vs non-attribution (BSD)

Pros:

- Enforcing some kind of attribution in prod could bring more popularity to project

Cons:

- Again, not everyone may be happy with such enforcement and such enforcement could play against us

## MIT vs Unlicense

All cons mentioned above are irrelevant if we just use MIT/Unlicense.

MIT is just an info about author without any restrictions, so It's mostly cool. What isn't cool is that most licenses require list of authors who modified source files. I don't really like keeping up-to-date list of contributors in copyright notice. That's getting messy very quickly.

So I propose using unlicense - at first line text below becomes copyrighted, then all rights are passed to everyone who's reading. Since there are no rights to be cleaimed, it's irrelevant who's listed as author. And there are really more important things to do than looking after all that license bs.

### LINKED ISSUE #48: Move util/manager-specific logging functions from log.c
- [x] logBitMap
- [x] logUCopList

### PR DESCRIPTION
**Pull request by tehKaiN**
fixes #43 
fixes #14 
fixes #48

### CODE REVIEW

File: showcase/src/test/copper.c [hunk 1]
```diff
@@ -136,19 +141,24 @@ void gsTestCopperLoop(void) {
...
-	if(uwY <= 30)
+	}
+	if(uwY <= 30) {
 		bDir = 1;
+	}
 
 	uwY += 2*bDir;
 
-	for(i = 0; i != 32; ++i)
+	for(i = 0; i != 32; ++i) {
```
Comment (approxit) [inline showcase/src/test/copper.c]: `!=` instead of `<=`?

File: src/ace/managers/timer.c [hunk 2]
```diff
@@ -162,5 +171,5 @@ void timerWaitUs(UWORD uwUsCnt) {
 	// timerGetPrec(): One tick equals: PAL - 0.40us, NTSC - 0.45us
 	ULONG ulStart = timerGetPrec();
 	UWORD uwTickCnt = uwUsCnt*2/5;
-	while(timerGetPrec() - ulStart < uwTickCnt);
+	while(timerGetPrec() - ulStart < uwTickCnt) {}
```
Comment (approxit) [inline src/ace/managers/timer.c]: Good example for clangformat. Guess adding something to empty code blocks will show in diff, which we agreed to mitigate. Should be in new lines?

File: src/ace/managers/viewport/scrollbuffer.c [hunk 3]
```diff
@@ -31,10 +35,15 @@ tScrollBufferManager *scrollBufferCreate(tVPort *pVPort, UBYTE ubMarginWidth, UW
 	vPortAddManager(pVPort, (tVpManager*)pManager);
 
 	// Find camera manager, create if not exists
-	if(!(pManager->pCameraManager = (tCameraManager*)vPortGetManager(pVPort, VPM_CAMERA)))
-		pManager->pCameraManager = cameraCreate(pVPort, 0, 0, uwBoundWidth, uwBoundHeight);
-	else
+	pManager->pCameraManager = (tCameraManager*)vPortGetManager(pVPort, VPM_CAMERA);
+	if(!pManager) {
```
Comment (approxit) [inline src/ace/managers/viewport/scrollbuffer.c]: Should it be `pManager->pCameraManager`?

File: src/ace/managers/timer.c (hunk 2 above)
Comment (tehKaiN) [reply src/ace/managers/timer.c]: I think that such change is significant enough to have it fully shown in diff. Also, the following code looks awful and doesn't look as done deliberately.

``` c
while(timerGetPrec() - ulStart < uwTickCnt) {
}
```

File: src/ace/managers/viewport/scrollbuffer.c (hunk 3 above)
Comment (tehKaiN) [reply src/ace/managers/viewport/scrollbuffer.c]: Thanks!

File: showcase/src/test/copper.c (hunk 1 above)
Comment (tehKaiN) [reply showcase/src/test/copper.c]: Thanks!

**Pull request merged**